mlflow/gluon/__init__.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, gluon_model):
     def predict(
         self,
         data,
-        params: Optional[Dict[str, Any]] = None,  # pylint: disable=unused-argument
```

Comment:
`predict` is an abstract method. `params` is unused but needs to be kept to avoid changing the function signature.

Reply:
Ruff has rules to detect unused arguments but not sure if we really need to enable them.